src/Components/AdvancedSearch/Internal/QueryBuilder/QueryBuilder.tsx

```diff
@@ -146,45 +148,51 @@ const QueryBuilder: React.FC<IQueryBuilderProps> = (props) => {
 
     return (
         <div className={classNames.root}>
-            <div className={classNames.headerGrid}>
-                {!(rows.length === 1) && (
+            <Stack tokens={{ childrenGap: 10 }}>
+                <div className={classNames.headerGrid}>
+                    {!(rows.length === 1) && (
```

Comment:
nit: instead of inverting the full condition can we do `rows.length !== 1`? Just easier to read IMO